akvo/rsr/views/py_reports.py

```diff
@@ -72,8 +74,10 @@ def render_organisation_projects_results_indicators_map_overview(request, org_id
         in projects
     ]
 
+    now = datetime.today()
+
     html = render_to_string(
-        'reports/organisation-projects-results-indicators-map-overview.html',
+        'reports/wp-organisation-projects-results-indicators-map-overview.html',
```

Comment:
I'd recommend deleting the other template, if we are not going to use it, and replacing it with the new template. 

Is it important to prefix these templates with `wp-` are they different from the other template that we write for `wkhtmltopdf`?  

Are we going to switch to using WeasyPrint for every report? Or do we still want to keep using `wkhtmltopdf`?

Reply:
These changes can come in a separate PR, though, if that works better.